matrix_sdk/src/client.rs

```diff
@@ -946,6 +950,40 @@ impl Client {
         self.store().get_room(room_id).and_then(|room| room::Left::new(self.clone(), room))
     }
 
+    /// Tries to decrypt a `AnyRoomEvent`. Returns unencrypted room event when
+    ///  decryption fails.
```

Comment:
There's an extra space here.

```suggestion
    /// decryption fails.
```